numba/decorators.py

```diff
@@ -77,6 +77,12 @@ def jit(signature_or_function=None, locals={}, target='cpu', cache=False, **opti
                 mode. Any arrays that the tight loop uses should be created
                 before the loop is entered. Default value is True.
 
+            error_model: str
+                The error-model affects divide-by-zero behavior.
```

Comment:
The original intent is to provide error-model control for different kinds of error, but only divide-by-zero is expose at this point.  When we have more error kinds, we should expose more finegrain control.

FYI, the underlying control is implemented by the two classes at: https://github.com/numba/numba/blob/e374e63178f2606279d297f208f6ae261196b0ba/numba/targets/callconv.py#L477-L496